src/components/ClusterManager.js

```diff
@@ -181,7 +182,10 @@ export default class ClusterManager extends PureComponent {
           h(GalaxyWarning),
           h(GalaxyLaunchButton, {
             app,
-            onClick: () => clearNotification(galaxyId)
+            onClick: () => {
+              clearNotification(galaxyId)
+              Ajax().Metrics.captureEvent(Events.applicationLaunch, { app: 'Galaxy' })
```

Comment:
Since we want to do the same thing in both places, we could add this to `GalaxyLaunchButton` instead (as long as we make sure it respects the passed `onClick` as well)

Reply:
oh I thought since the instances defined their own `onClick`s adding this to the definition would override that, I will look into moving it, thanks!